pkg/operators/wasm/maps.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2024 The Inspektor Gadget authors
```

Comment:
Once agreed on implementation, please remember to update [the API documentation](https://www.inspektor-gadget.io/docs/latest/gadget-devel/gadget-wasm-api-raw#api). Thanks!